oi/keymap.py

```diff
@@ -5,11 +5,17 @@
     DefaultButton,
 )
 
+controllerDRIVER = XBoxController
+controllerOPERATOR = XBoxController
 
 class Controllers:
-    DRIVER: int
-    OPERATOR: int
+    DRIVER = 0
```

Comment:
```suggestion
    DRIVER: int = 0
```
I would suggest keeping these types, at least for larger objects like 'SwerveNode'. They lead to better readability and IDE completion. You can define typed variables like above ^